src/pages/PdfViewerPage/PdfViewerPage.tsx

```diff
@@ -28,7 +29,10 @@ const PdfViewerPage: FC<PdfViewerPageProps> = ({
       className={cs("sa-pdf-viewer-page", className)}
       data-testid={dataTestIDs.root}
     >
-      <div>Pdf Viewer Page</div>
+      <div>
+        <Header />
```

Comment:
Instead of adding the Header in each of the pages, it may be better to add it in the `<Router>` component in `App.tsx`